torchsnapshot/storage_plugin.py

```diff
@@ -41,6 +41,10 @@ def url_to_storage_plugin(url_path: str) -> StoragePlugin:
         from torchsnapshot.storage_plugins.gcs import GCSStoragePlugin
 
         return GCSStoragePlugin(root=path)
+    elif protocol.startswith("fsspec-"):
+        from torchsnapshot.storage_plugins.fsspec import FSSpecStoragePlugin
+
+        return FSSpecStoragePlugin(root=path)
```

Comment:
this might run into conflicts this PR which adds the storage options to the storage plugin constructor: https://github.com/pytorch/torchsnapshot/pull/108

Reply:
yes, I will fix the conflict when the pr #108 be merged to main.